docs/DeveloperGuide.md

```diff
@@ -826,34 +880,47 @@ Similar to <u>deleting an elderly (UC3)</u> but only deleting an elderly's Nok d
     * 2e1. NurseyBook shows an error message.
 
       Use case resumes at step 1.
-    
-**UC13: Mark a task as complete**
+* *a. At any time, user requests to <u>view help (<a href="#uc19-viewing-help">UC19</a>)</u>.
+
+##### UC16: Mark a task as complete
+
+Similar to <u>deleting a task (<a href="#uc14-delete-a-task">UC14</a>)</u> but marking a task as done instead. The `doneTask` command success message will appear instead.
+
+##### UC17: Find a task
+
+Similar to <u>finding an elderly (<a href="#uc5-find-an-elderly">UC5</a>)</u> but finding a task instead. A task will be shown (as part of the filtered task list) if its description contains the entered keywords.
+
+##### UC18: View the schedule on a day
 
 **MSS**
 
-1. User requests to <u>list tasks (UC10)</u>
-2. User requests to mark a specific task in the list as complete
-3. NurseyBook marks task as complete
-4. NurseyBook shows updated list of tasks and command success message.
+1. User requests to view his/her schedule on a date.
+2. NurseyBook shows the list of tasks scheduled to happen on the date.
 
-    Use case ends.
+   Use case ends.
 
 **Extensions**
+* 1a. The given date is in an invalid format.
+    * 1a1. NurseyBook shows an error message.
 
-* 2a. The given index is invalid.
+      Use case resumes at step 1.
+* 2a. There are no tasks scheduled on that date.
 
-    * 2a1. NurseyBook shows an error message.
+     Use case ends.
```

Comment:
Perhaps an extension on the date being outside the range of 3 months into the future can be added?